pyaedt/emit.py

```diff
@@ -154,6 +159,31 @@ def __init__(
 
             self.__emit_api_enabled = True
 
+        # Update modeler IDs now, after units are initialized.
+        # self.modeler.components.refresh_all_ids()
```

Comment:
```suggestion
```
Can we remove these?

Reply:
@jsalant22 - Yes! Good catch. Thank you.